.github/workflows/staging_queryweaver(staging).yml

```
@@ -0,0 +1,71 @@
+# Docs for the Azure Web Apps Deploy action: https://github.com/Azure/webapps-deploy
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)